LibGUI/GScrollBar.cpp

```diff
@@ -200,10 +200,12 @@ void GScrollBar::paint_event(GPaintEvent& event)
     painter.fill_rect(rect(), Color::from_rgb(0xd6d2ce));
 
     StylePainter::paint_button(painter, decrement_button_rect(), ButtonStyle::Normal, false, m_hovered_component == Component::DecrementButton);
-    painter.draw_bitmap(decrement_button_rect().location().translated(3, 3), orientation() == Orientation::Vertical ? *s_up_arrow_bitmap : *s_left_arrow_bitmap, has_scrubber() ? Color::Black : Color::MidGray);
-
     StylePainter::paint_button(painter, increment_button_rect(), ButtonStyle::Normal, false, m_hovered_component == Component::IncrementButton);
+    
+    if(orientation() == Orientation::Vertical ? height() > (default_button_size()) : width() > (default_button_size())){
```

Comment:
Please conform to the project coding style. You can use `clang-format` (please use at least version 7, preferably 8) on the source file and it will automatically adjust indentation, etc.